internal/doctl/mylib.go

```diff
@@ -0,0 +1,182 @@
+package mylib
```

Comment:
change the package name to `doctl` and rename this file to `doctl.go`?

Reply:
done